client/lib/application/src/application-detail/application-detail.jsx

```diff
@@ -44,6 +45,14 @@ export default function ApplicationDetail({applicationId, application, versions,
                         className='btn btn-primary'>
                         <Icon name='list' /> Versions
                     </Link>
+                    {ENV_DEVELOPMENT ?
+                        <Link
+                            to={Routes.lifecycle(LINK_PARAMS)}
+                            className='btn btn-primary'>
+                            <Icon name='list' /> Lifecycle
```

Comment:
Sure. This just pure for having "something" as a sub screen for further work. I will change the icon as soon as I introduce actual components for application lifecycle. Deal?